.circleci/config.yml

```diff
@@ -2254,6 +2254,13 @@ jobs:
             python ../build_package.py deb
             python ../build_package.py rpm
 
+            dpkg -X "$(ls *.deb)" /tmp/deb_package
+            warnings=$(dpkg-parsechangelog --file /tmp/deb_package/usr/share/doc/scalyr-agent-2/changelog.gz 2>&1 1>/dev/null | grep warning || true)
+            if [[ -n "${warnings}" ]]; then
+              echo "The debian package changelog is built with warnings."
```

Comment:
And here we should also echo the actual warnings + some friendly text along the lines of "Text shouldn't be split / wrapped across multiple lines, etc.".